package.json

```diff
@@ -35,5 +35,8 @@
     "execa": "^2.0.3",
     "node-emoji": "^1.10.0",
     "ora": "^3.4.0"
+  },
+  "peerDependencies": {
+    "lerna": "^3.22.1"
```

Comment:
I believe this version should be `3.x`.  We want to support everything except a major version of lerna that could change the API.